### DISCUSSION TITLE
Art Contributors 2023

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1935] [General]
The purpose of this document is to explore the vision and plan to improve the art in Sunflower Land.

We are onboarding more artists as contractors at Sunflower Land. Keeping up to date with all of the art submissions was becoming overwhelming for the core team - over the past few weeks I have lost the close connection and some pieces of amazing art have slipped by.

## 2022 Artist Survey Results

**Freedom + Collaboration was the main theme of the community**

- “I love the freedom of expression of the community and the exchange of ideas”
- “How open for improvements and feedback each artist”
- “We can grow together as artists and help each other”

**Challenges**

- Lack of planning around upcoming events - “for example if there will be a valentine's event it should be in planning already”
- Structure in Discord is not clear - How do you get an Artist badge?
- “Not enough critics before pushing the art to mainnet”
- Lack of guidance on art - “Docs are a good starting point, but preferably someone should be able to provide constant feedback to help nurture new artists”
- Lack of communication with original artists - If changes are suggested, the original artist should be given the opportunity first to make the adjustments. 
- “So many art requests, then nothing gets picked or moved forward. it can end up feeling like wasted time”

## Art Community 2023

The goal for 2023 is to provide more structure & transparency to help existing and new comers contribute art to our beloved community and game.

Thought Farm alongside the contractor's aim is to alleviate the administrative/chore related work so community members have the freedom & clarity to focus on a range of creative initiatives.

The easiest way to scale art in a game is a closed door approach and done 100% in-house. However, the vision for Sunflower Land is to continue down the community built path - It has its challenges, but we are strong believers in the power of a community and what it can do. In order to continue this community approach we are making some proactive changes.

### Structure

To help us organise community contributions, we will be improving our Discord structure, cadence and transparency of work that needs to be done.

**Public Design Roadmap** - We will publicise a design roadmap which includes important designs & ideas that need to be worked on. This includes upcoming features, decorations, special events etc. Gabi will help organise this public design roadmap.

**Weekly community artist meeting**

Once a week, we will host a Discord voice chat where the community can discuss ideas, priorities and upcoming work. A core team member and art contractors will join these meetings to help run them. Here we will talk about the public design roadmap and take on suggestions. It will be a great time to volunteer for tasks yourself. This is also the time to go over community ideas for in-game items and Bumpkins and add them to our roadmap. We will also discuss the best contributions from the week.

Output - Meeting notes for those unavailable to join
Time - TBD

**Live Twitch Streams**

Gabi and the core team will begin design focussed Twitch streams. The aim of these streams are to provide feedback on community designs and also do live pixel art. Here we will talk about Sunflower Land aesthetics, how to get started, colour palettes etc.

**Discord roles**

`Art Contributor` - Contribute 3 pieces of art to the game. Art contributors will get some beautiful ‘artist’ cosmetics and decorations to show off in the game! This group will be able to take part in voting on upcoming competitions & proposals.

`Art Leaders` - Art leaders are a smaller subset of individuals whose focus will be giving feedback, guiding the community and improving the processes. This group will be much smaller (5-6 people) and reserved for those who not only produce art well aligned with the game’s aesthetics, but also are extremely active and helpful in the community. Members will come and go based on their activity and voted in by other members of the community.

### Contracted Roles

This was an incredibly difficult decision to make - there were 3-4 artists who were all outstanding candidates for this role. There are clearly members in this community who are doing incredible art & leading initiatives. As we grow, we are hoping to open up more opportunities for these members.

For those that submitted applications we are extremely grateful and saddened that we couldn’t hire everyone. We look forward to more collaborations and exciting submissions from you in 2023!

**Why are we doing contractor roles?**

Unsurprisingly, a significant amount of admin work has arisen as our community has grown. A tremendous amount of tasks now exist which have shifted away from purely design, brainstorming and creativity. There are new responsibilities, chores & timelines that fall under traditional working roles.

This work includes meetings, creating design briefs, checking for plagiarism, organising Github requests, ensuring we meet deadlines, streaming educational videos etc.

As we grow as a community, these problems are only going to intensify. To be proactive, we are onboarding artists to help out in these roles

The goal of Sunflower Land contractors first and foremost is to serve the community. While they will be contributing art, the main goal is to focus on improving community contributions and helping get them into the game.

**Art Design Lead**

Gabi, will be stepping up into the role of Art Design Lead. She has done a tremendous job producing high quality art works and community templates, but also shown her ability to teach and collaborate with her live Discord pixel art streams. We will be expecting more active management of art proposals and activity in Discord.

She will be joining full-time and will be taking on some crucial responsibilities:

- Managing & creating design briefs
-Leading/working with the community for large scale projects
 & features
- Improving art guidelines and processes
- Live Twitch education streams

**Bumpkin Designer**

LittleEins, will be taking on the role of Bumpkin Designer. They have done an outstanding job not only with Bumpkin submissions, but also tweaking the base Bumpkin templates & providing female body shapes. One of the stand out characteristics from LittleEins was their ability to try different Bumpkin styles and their empathy for the community.

They will be joining part time and taking on the following roles:

-Summarising and highlighting the best community designs each week
- Providing feedback on Discord + Github Designs. Ensuring submissions follow guidelines and aesthetics
- Creating and highlighting ideas from the community for Bumpkins
- Working with design briefs and community members to create Bumpkin art
- Ensuring no plagiarism on submitted art

**Map & Decoration Designer**

Oniel will be taking on the role of map & decoration designer. By now everyone knows their awesome art and involvement in the community. One of the stand out characteristics from Oniel was their ability to consistent contribute to Github art requests and their iterations they have made on previous art in the game - They will be pushing us to not only create new art, but to collaborate improve existing designs.

They will be joining part time and taking on the following roles:

- Summarising and highlighting the best community designs each week
- Providing feedback on Discord + Github Designs. Ensuring we achieve a consistent in-game aesthetic with colours + dimensions
- Creating and highlighting ideas from the community for map ideas, items, quests and decorative items
- Ensuring no plagiarism on submitted art

### Noteworthy Points for 2023

**Commissioned pieces**

We want to keep the Open Source mentality to contributing and don’t want us turning in a ‘paid’ mentality. However, it is evident that some individuals are doing large projects of work including promotional length videos, in depth asset packs, branding sets etc. 

In 2023 we will provide more opportunities for community members to take on these commissioned pieces of work.

We will reach out to artists in the community when these opportunities arise.

For typical art requests/submissions (decorations, Bumpkin wearables etc.) we will continue down the path of no monetary commission. The key reason is so these contributions keep our contributive/open source ethos and do not become a money grab.

**Community Events**

The Halloween and Christmas events were huge successes in the game! It is incredible to see the effort and creativity that goes into these initiatives, especially since they have very little core team influence.

Our role as core team is to ensure that these community led events continue to make it into the game.

During the 2023 Christmas season we witnessed the success of the donation model. We will continue this path for the following events. In 2023 these ‘community events’ will be more planned and given a lot of time’s notice to prepare.

**Collaboration / Improvements**

The community has set a new benchmark for art quality since the launch of Sunflower Isles. We expect that all art in the game is dynamic and expect community members to make improvements on existing art.

Just a reminder that all art submitted to the project becomes available to other artists to iterate freely on. However, it is recommended to first message the original artists with some suggestions first and see if they want to try & make the improvements.

**Comment by LeShykun** [comment]
Looking toward 2023 ! LFG!!!

**Comment by Myth727** [comment]
@LittleEins congrats mate on your role! 🔥

**Reply by LittleEins** [reply]
Thank you!!!
